src/lua/thread.rs

```diff
@@ -130,7 +130,13 @@ pub fn init() {
     let (use_config, maybe_init_file) = init_path::get_config();
     if use_config {
         match maybe_init_file {
-            Ok(init_file) => {
+            Ok((init_dir, init_file)) => {
+                if init_dir.components().next().is_some() {
+                    // Add the config directory to the package path.
+                    let mut package: hlua::LuaTable<_> = lua.get("package").unwrap();
+                    let paths: String = package.get("path").unwrap();
```

Comment:
While you're right that these can't ever happen, given a valid Lua environment, just in case can you change these to `expects` that say something to the effect of `"package not defined in lua"` and `"package.path not defined in lua"`. Makes it slightly easier to figure out the error when inevitably that user who uses a weird lua sends me a bug report ;)